File: RobotiumDemo/RobotiumDemo/app/src/main/java/org/lynxz/robotiumdemoapp/MainActivity.java
package org.lynxz.robotiumdemoapp;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button btnHi = (Button) findViewById(R.id.btn_hello);
        btnHi.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(MainActivity.this, "nice to meet you", Toast.LENGTH_LONG).show();
            }
        });
    }
}

File: ImageLoaderLibsDemo/app/src/main/java/org/lynxz/imageloaderlibsdemo/MainActivity.kt
package org.lynxz.imageloaderlibsdemo

import android.net.Uri
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.facebook.drawee.view.GenericDraweeView
import kotlinx.android.synthetic.main.activity_main.*


class MainActivity : AppCompatActivity() {

    val imgUrl = "https://www.fresco-cn.org/static/logo.png"


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        loadByFresco(Uri.parse(imgUrl), sdv_demo)
        loadByFresco(Uri.parse("res://org.lynxz.imageloaderlibsdemo/" + R.mipmap.ic_launcher), sdv_demo_local)
    }

    /**
     * 通过fresco加载网络图片
     * */
    private fun loadByFresco(uri: Uri, targetView: GenericDraweeView) {
        targetView.setImageURI(uri)
    }
}
